### PR TITLE
Cross table improvements

### PR DESCRIPTION
**Pull request by gijskant**
- Show scrollbar when cross table size exceeds panel width.
- Only enable adding cross table when none is present.
- Remove 'clear cache' button when Fractalis is not enabled.
- Do not change row constraints when loading
- Update cross table layout.

Fixes [TMT-1007](https://jira.thehyve.nl/browse/TMT-1007).

### CODE REVIEW

File: src/app/modules/gb-analysis-module/cross-table-components/gb-cross-table/gb-cross-table.component.ts [hunk 1]
```diff
@@ -23,6 +24,8 @@ import {FormatHelper} from '../../../../utilities/format-helper';
 })
 export class GbCrossTableComponent implements OnInit {
 
+  AxisType = AxisType;
```
Comment (ewelinagr) [inline src/app/modules/gb-analysis-module/cross-table-components/gb-cross-table/gb-cross-table.component.ts]: Why is this needed?

Comment (gijskant) [reply src/app/modules/gb-analysis-module/cross-table-components/gb-cross-table/gb-cross-table.component.ts]: Otherwise you cannot use the type in the html template.
